### DISCUSSION TITLE
Change language to German

### DISCUSSION THREAD

**Discussion by cgeheim** [immich-app/immich#8519] [Q&A] [answered]
Is there a way  to change language to German in the server application and the iOS app?

I do not find a way and no description 

Thank you in advance 

Claus

**Comment by bo0tzz** [comment, marked as answer]
The web client doesn't have translations yet. The app will follow your system language.

**Reply by cgeheim** [reply]
Thank you very much

**Comment by cgeheim** [comment]
Thanks
